src/Features/Auth/authSlice.js

```
@@ -44,7 +44,7 @@ export const login = createAsyncThunk("auth/login", async (form, thunkApi) => {
 });
 
 export const update = createAsyncThunk("auth/update", async (data, thunkApi) => {
-	const { authToken: token, localData: form } = data;
+	const { localData: form } = data;
 	const user = jwtDecode(token);
```

ESLint

[error] 'token' is not defined.

(no-undef)